AmplifyPlugins/API/AWSAPICategoryPluginIntegrationTests/REST/RESTWithUserPoolIntegrationTests/RESTWithUserPoolIntegrationTests.swift

```diff
@@ -13,25 +13,15 @@ import AmplifyPlugins
 @testable import AmplifyTestCommon
 
 class RESTWithUserPoolIntegrationTests: XCTestCase {
-    struct User {
-        let username: String
-        let password: String
-    }
+    
     let amplifyConfigurationFile = "testconfiguration/RESTWithUserPoolIntegrationTests-amplifyconfiguration"
-    let credentialsFile = "testconfiguration/RESTWithUserPoolIntegrationTests-credentials"
```

Comment:
Here and other places, there is a README associated with each integration test, if we no longer need the credentials file, please update the README to reflec that